spacemacs/packages.el

```diff
@@ -136,6 +137,35 @@ which require an initialization must be listed explicitly in the list.")
       (setq ace-jump-mode-scope 'global)
       (evil-leader/set-key "`" 'ace-jump-mode-pop-mark))))
 
+(defun spacemacs/init-ace-link ()
+  (use-package ace-link
+    :defer t
+    :init (ace-link-setup-default)
+    :config
+    (progn
+      (defvar spacemacs--link-pattern "~?/.+\\|\s\\[")
+
+      (defun spacemacs//collect-spacemacs-buffer-links ()
+        (let ((end (window-end))
+              points)
+          (save-excursion
+            (goto-char (window-start))
+            (while (re-search-forward spacemacs--link-pattern end t)
+              (push (+ (match-beginning 0) 1) points))
+            (nreverse points))))
+
+      (defun spacemacs//ace-buffer-links ()
```

Comment:
Should be `/` instead of `//` since it is a "public" (in this case interactive) function.